core/controlplane/config/templates/cloud-config-controller

```diff
@@ -9,6 +9,20 @@ coreos:
     etcd_keyfile: /etc/kubernetes/ssl/etcd-client-key.pem
 
   units:
+{{- range $thisunit := .Controller.CustomUnits}}
```

Comment:
nit: I prefer just `$u` rather than `$thisunit` as per the golang convention of using a single character for a loop variable.